db/schema.rb

```diff
@@ -10,13 +10,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_11_28_223151) do
+ActiveRecord::Schema.define(version: 2021_02_14_220111) do
 
   create_table "adverts", force: :cascade do |t|
     t.string "title"
     t.string "address"
-    t.integer "price"
-    t.string "price_arranged"
+    t.decimal "price"
+    t.string "negotiated_price"
```

Comment:
it's a bit weird to keep price as `string`, so let's create one more migration and convert `negotiated_price` field to `decimal` too.

Reply:
And then please add test to check if you can use `negotiated_price` field in tests, and save float amount like `9.99`